larray_editor/view.py

```diff
@@ -554,7 +553,7 @@ def paste(self):
 
     def plot(self):
         if not matplotlib_present:
-            raise Exception("plot() is not available because matplotlib is not installed")
+            QMessageBox.critical(self, "Error", "plot() is not available because matplotlib is not installed")
```

Comment:
It never happens if larray-editor is installed via conda. 

matplotlib is not a dependency in the setup.py 
The reason for that is, if we add dependencies in setup.py, we must use 
 > build:
 >    number: 0
 >    script: python setup.py install --single-version-externally-managed --record record.txt
in the meta.yml file of conda package. But, in this case, menuinst is disabled...